.cloud-build/execute_changed_notebooks_helper.py

```diff
@@ -503,10 +505,13 @@ def process_and_execute_notebooks(
 
         print(f"Found {len(notebooks)} modified notebooks: {notebooks}")
 
+        rate_limit = RateLimit(max_count=concurrent_notebooks, per=60, greedy=False)
```

Comment:
Still don't need this as mentioned in the previous comment.

Reply:
I must've missed seeing the comment

Reply:
done